.github/workflows/ktlint.yml

```
@@ -60,22 +60,8 @@ jobs:
                     --reporter=checkstyle,output=build/ktlint-checkstyle.xml \
                     "${files[@]}"
 
-            - name: Install reviewdog
+            - name: Post inline review comments
               if: always() && steps.changed.outputs.count != '0'
-              uses: reviewdog/action-setup@v1
+              uses: actions/github-script@v7
```

actionlint (1.7.12)

[error] missing input "script" which is required by action "actions/github-script@v7". all required inputs are "script"

(action)